consensus/src/dag/types.rs

```diff
@@ -571,7 +586,21 @@ impl CertifiedAck {
     }
 }
 
-impl BroadcastStatus<DAGMessage> for CertificateAckState {
+impl From<CertifiedAck> for DAGRpcResult {
```

Comment:
feels belonging to the beautiful macro you wrote?

Reply:
This one won't fit the macro because DAGRpcResult is a wrapper and also a result type. slightly more complicated.